update-workflow

```diff
@@ -4,6 +4,8 @@ set -e
 enrolling_root=$(git rev-parse --show-toplevel)
 config_root=$enrolling_root/conjur-project-config
 
+export default_git_branch="$(git remote show origin | grep 'HEAD branch' | cut -d' ' -f5)"
+
```

Comment:
Formatting + slightly simpler:
```bash
export default_git_branch="$(
  git remote show origin | awk '/HEAD branch/{print $NF}'
)"
```

Reply:
There's a lot of info that is output from `git remote show origin` for for just one line that's needed.  This should do the same thing but be substantially faster and is only processing output of one line

```bash
export default_git_branch="$(git rev-parse --abbrev-ref origin/HEAD | awk -F'/' '{print $NF}')"
```

Reply:
Just out of curiosity, why are we worried about the efficiency here? Is Bash really that inefficient that we need to minimize everything as script size grows? Or is it just from a "we want to make sure everything we do is as efficient as possible" perspective? I'm relatively new to large scale bash scripting so I'm just trying to understand the thinking